bindings/nodejs/lib/types/block/payload/signed_transaction/unlock.ts

```diff
@@ -139,6 +147,37 @@ class NftUnlock extends Unlock {
     }
 }
 
+/**
+ * Unlocks a MultiAddress with a list of other unlocks.
+ */
+class MultiUnlock extends Unlock {
+    /**
+     * The inner unlocks.
+     */
+    @Type(() => Unlock, {
+        // @ts-ignore:next-line: no-use-before-declare
+        discriminator: UnlockDiscriminator,
```

Comment:
Well... technically this should be a new discriminator that excludes multi unlock? Similar to what we did for RestrictedAddress. Or do we just not care and it will fail rust side?

Reply:
Currently we don't care for all the unlock conditions and features of outputs, but I can also change if we prefer that

Reply:
I think I'm fine with it failing in Rust tbh, we can't put all syntactic rules in bindings anyway 🤷🏻